dataStorage/storage.js

```diff
@@ -24,7 +24,7 @@ function encode(idOrKey) {
     return idOrKey.replace(/_/g, '##-##');
 }
 
-async function loadAsync(model, id) {
+async function loadAsync(model, id, update) {
```

Comment:
Can you please add a comment for this method, explaining how the parameters are used? thansk!